### PR TITLE
Multireddit quicklinks menu

### PR DESCRIPTION
**Pull request by andytuba**
screen shot 2016-04-10 at 8 08 33 pm

### CODE REVIEW

File: lib/modules/multiredditNavbar.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+			.reduce((prev, curr) => (curr ? prev.add(curr) : prev), $())
+			.add(populateSectionItem('.', [
+				`<i>${module.moduleName}</i>`,
+				modules['settingsNavigation'].makeUrlHash(moduleID, 'sectionMenu')
+			]))
+	];
+
+	function populateSectionItem(baseUrl, link) {
+		if (!(link && link.length >= 2)) {
+			return;
```
Comment (erikdesjardins) [inline lib/modules/multiredditNavbar.js]: Errors from the `consistent-return` lint can be a bit misleading; it wants all or none of the return statements to return a value (`undefined` is allowed, as long as it's explicit).

File: lib/modules/multiredditNavbar.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+addModule('multiredditNavbar', (module, moduleID) => {
+	module.moduleName = 'Multireddit Navigation';
+	module.description = 'Enhance the navigation bar shown on the left side of the frontpage.';
+	module.category = 'Subreddits';
+
+	module.includes = [
+		RESUtils.regexes.frontpage
+	];
+
+	Object.assign(module.options, {
```
Comment (erikdesjardins) [inline lib/modules/multiredditNavbar.js]: This should be identical to `module.options = {` ?

File: lib/modules/multiredditNavbar.js (hunk 1 above)
Comment (andytuba) [inline lib/modules/multiredditNavbar.js]: `return false` seems like a reasonable thing for the linter to ask for.

File: lib/modules/multiredditNavbar.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+				offsetHeight: 1,
+				bottomPadding: 0,
+			})
+			.populateWith(() => populateSectionMenu(link.href))
+			.begin();
+	}
+
+	const populateSectionMenu = (baseUrl) => [
+		module.options.sectionLinks.value
+			.map((...args) => populateSectionItem(baseUrl, ...args))
```
Comment (erikdesjardins) [inline lib/modules/multiredditNavbar.js]: `.map(link => populateSectionItem(baseUrl, link))` ?

Comment (andytuba) [inline lib/modules/multiredditNavbar.js]: what, being explicit and performant? that's crazy talk.

**Pull request merged**